dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,4 +21,16 @@ def template
 
     render template: "projects/#{params[:template]}", layout: nil
   end
+
+  def new
```

Comment:
Ah that makes way more sense.  But how do we specify which level maps to /p/artist and /p/playlab?  There can only be one "master" project .level file.